TwilioVerifySDK/TwilioVerify/Sources/Models/Factor.swift

```diff
@@ -38,6 +38,8 @@ public protocol Factor {
   var type: FactorType { get }
   ///Indicates the creation date of the Factor.
   var createdAt: Date { get }
+  ///Custom metadata associated with the factor when created. This is added by the Device/SDK directly to allow for the inclusion of device information.
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/9cbd540c6135af3d1641ad17aef5ccc5e04d146c/TwilioVerifySDK/TwilioVerify/Sources/Models/Factor.swift#L41"></span>Prefer at least one space after slashes for comments.<br />
<code>comment_spacing</code> <code>Factor.swift:41</code></td>
    </tr>
  </tbody>
</table>